### PR TITLE
Update .pre-commit-config.yaml

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ repos:
       - id: end-of-file-fixer
       - id: check-docstring-first
       - id: check-json
+        exclude: 'asv_bench/asv.conf.json'
       - id: check-yaml
-      - id: double-quote-string-fixer
```
Comment (maxrjones) [inline .pre-commit-config.yaml:13]: If we remove this check (which I think would be good), we should enable the default string normalization setting for *Black*:
https://github.com/carbonplan/python-project-template/blob/0dc5bd5b133f931297e694d6f395c1d05f4d0569/pyproject.toml#L4

File: pyproject.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [tool.black]
 line-length = 100
 target-version = ['py38']
-skip-string-normalization = true
+skip-string-normalization = false
```
Comment (maxrjones) [inline pyproject.toml]:
```suggestion
```

**Review by maxrjones — APPROVED** [review]
Nice improvement! just made one small suggestion

**Comment by norlandrhagen** [comment]
> Nice improvement! just made one small suggestion

oops haha

**Pull request merged**